src/main/java/org/fundacionjala/coding/melvi/MovieRefactor/ReportGenAbstract.java

```diff
@@ -0,0 +1,16 @@
+package org.fundacionjala.coding.melvi.MovieRefactor;
+
+import java.util.Enumeration;
+
+/**
+ *
+ */
+public abstract class ReportGenAbstract {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "ReportGenAbstract" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)